homework_5.py

```diff
@@ -0,0 +1,32 @@
+"""There is my homework #5"""
+# HW 5
+# 1. Заменить символ “#” на символ “/” в строке
+# 'www.my_site.com#about'
+site_url = 'www.my_site.com#about'
+site_url_2 = list(site_url)
```

Comment:
you do not need to convert to list and back to string just for replacement 
use `replace()` method instead

Reply:
Changes have been done